source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/CalculationJobType.java

```diff
@@ -59,24 +63,41 @@ public enum CalculationJobType {
   IN_COMBINATION_PROCESS_CONTRIBUTION(new HashSet<>(Arrays.asList(SituationType.PROPOSED)),
       new HashSet<>(
           Arrays.asList(SituationType.REFERENCE, SituationType.NETTING, SituationType.COMBINATION_REFERENCE, SituationType.COMBINATION_PROPOSED)),
-      new HashSet<>(Arrays.asList(SituationType.COMBINATION_PROPOSED, SituationType.COMBINATION_REFERENCE))),
+      new HashSet<>(Arrays.asList(SituationType.COMBINATION_PROPOSED, SituationType.COMBINATION_REFERENCE)),
+      1, 6),
 
   /**
    * Only 1 Reference scenario
    *
    * NO other scenario types
    */
-  DEPOSITION_SUM(new HashSet<>(Arrays.asList(SituationType.REFERENCE)), new HashSet<>(), new HashSet<>());
+  DEPOSITION_SUM(new HashSet<>(Arrays.asList(SituationType.REFERENCE)),
+      new HashSet<>(),
+      new HashSet<>(),
+      1, 1),
+
+  /**
+   * One scenario (all types allowed)
+   */
+  SINGLE_SCENARIO(new HashSet<>(),
+      new HashSet<>(Arrays.asList(SituationType.values())),
+      new HashSet<>(),
+      1, 1);
 
   private final @JsProperty Set<SituationType> requiredSituationTypes;
   private final @JsProperty Set<SituationType> optionalSituationTypes;
   private final @JsProperty Set<SituationType> pluralSituationTypes;
 
+  private final int minimumNumberOfSituations;
+  private final int maximumNumberOfSituations;
+
   CalculationJobType(final Set<SituationType> requiredSituationTypes, final Set<SituationType> optionalSituationTypes,
-      final Set<SituationType> multipleSituationsAllowed) {
+      final Set<SituationType> multipleSituationsAllowed, final int minimumNumberOfSituations, final int maximumNumberOfSituations) {
     this.requiredSituationTypes = requiredSituationTypes;
     this.optionalSituationTypes = optionalSituationTypes;
     this.pluralSituationTypes = multipleSituationsAllowed;
+    this.minimumNumberOfSituations = minimumNumberOfSituations;
+    this.maximumNumberOfSituations = maximumNumberOfSituations;
```

Comment:
Not sure what the point of these are, and if it makes sense to keep them in this enum: minimum is always 1, and maximum isn't as strict for things like `IN_COMBINATION_PROCESS_CONTRIBUTION` is it? Isn't it usually a situation type that should be at most 1 in a calculation job?